gradle/libs.versions.toml

```
@@ -29,6 +30,15 @@ spring-boot-three = { id = "org.springframework.boot", version.ref = "springThre
 spring-dependency-management = { id = "io.spring.dependency-management", version = "1.0.11.RELEASE" }
 
 [libraries]
+androidx-activity-compose = { module = "androidx.activity:activity-compose", version = "1.6.3" }
+androidx-compose-foundation = { module = "androidx.compose.foundation:foundation", version = "1.6.3" }
+androidx-compose-foundation-layout = { module = "androidx.compose.foundation:foundation-layout", version = "1.6.3" }
+androidx-compose-material3 = { module = "androidx.compose.material3:material3", version = "1.2.1" }
+androidx-compose-ui = { module = "androidx.compose.ui:ui", version = "1.6.3" }
+androidx-compose-ui-replay = { module = "androidx.compose.ui:ui", version = "1.5.0" }
```

```suggestion
androidx-compose-ui-replay = { module = "androidx.compose.ui:ui", version = "1.5.0" } # Note: don't change without testing forwards compatibility
```
would love to keep the comment :)